pipes/reddit-auto-posts/components/pipe.tsx

```diff
@@ -0,0 +1,297 @@
+              settings.customSettings?.["reddit-auto-posts"]?.interval / 60)} 
+          </div>
+        </div>
+        <div className="space-y-3">
+          <Label htmlFor="pageSize">page size </Label>
+          <span className="text-[13px] text-muted-foreground">&nbsp;&nbsp;number of records to retrieve per page for extraction, considering LLM context limits</span>
+          <Input
+            id="pageSize"
+            name="pageSize"
+            type="number"
+            defaultValue={settings.customSettings?.["reddit-auto-posts"]?.pageSize || 100}
+            placeholder="size of page"
+          />
+        </div>
+        <div className="space-y-3">
+          <Label htmlFor="summaryFrequency">summary frequency </Label>
+          <span className="text-[13px] text-muted-foreground">&nbsp;&nbsp;email frequency: &apos;daily&apos; at email time or &apos;hourly:X&apos;(e.g.
+            &apos;hourly:4&apos; for every 4 hrs).</span>
+          <Input
+            id="summaryFrequency"
+            name="summaryFrequency"
+            defaultValue={settings.customSettings?.["reddit-auto-posts"]?.summaryFrequency || "daily"}
+            placeholder="frequency of summary emails: 'daily' for once a day at emailTime, or 'hourly:X' for every X hours (e.g., 'hourly:4' for every 4 hours)"
+          />
+        </div>
+        <div className="space-y-2">
+          <Label htmlFor="emailTime">email time </Label>
+          <span className="text-[13px] text-muted-foreground">&nbsp;&nbsp;time to send daily summary email (used only if summaryFrequency is &apos;daily&apos;)</span>
+          <Input
+            id="emailTime"
+            name="emailTime"
+            type="time"
+            defaultValue={settings.customSettings?.["reddit-auto-posts"]?.emailTime || "11:00"}
+            placeholder="time to send daily summary email (used only if summaryFrequency is 'daily')"
+          />
+        </div>
+        <div className="space-y-2">
+          <Label htmlFor="emailAddress">email address </Label>
+          <span className="text-[13px] text-muted-foreground">&nbsp;&nbsp;email address to send the daily summary to: (eg. me@mail.com)</span>
+          <Input
+            id="emailAddress"
+            name="emailAddress"
+            type="email"
+            defaultValue={settings.customSettings?.["reddit-auto-posts"]?.emailAddress || ""}
+            placeholder="email address"
+          />
+        </div>
+        <div className="space-y-3 relative items-center">
+          <Label htmlFor="emailPassword">email app specific password </Label>
+          <span className="text-[13px] text-muted-foreground">&nbsp;&nbsp;app specific password for your gmail account, you can find it
+            <a target="_blank" onClick={() => window.open("https://support.google.com/accounts/answer/185833?hl=en")} className="hover:underline text-sky-700"> here</a></span>
```

Comment:
why not normal href

https://www.w3schools.com/tags/tag_a.asp

with blank to open in new nab

Reply:
atm does not work